cmake/CorrosionGenerator.cmake

```diff
@@ -206,4 +206,13 @@ function(_generator_add_cargo_targets)
     else()
         message(DEBUG "Corrosion created the following CMake targets: ${curr_created_targets}")
     endif()
+
+    foreach(target_name ${created_targets})
+        foreach(output_var RUNTIME_OUTPUT_DIRECTORY PDB_OUTPUT_DIRECTORY ARCHIVE_OUTPUT_DIRECTORY LIBRARY_OUTPUT_DIRECTORY PDB_OUTPUT_DIRECTORY)
+            get_target_property(output_dir ${target_name} "${output_var}")
+            if (NOT output_dir AND DEFINED "CMAKE_${output_var}")
+                set_property(TARGET ${target_name} PROPERTY ${output_var} "${CMAKE_${output_var}}")
```

Comment:
We would probably also want to add a loop which does this for all `CMAKE_CONFIGURATION_TYPES` and the respective `_<CONFIG>` properties.

Reply:
Ah yes! That makes sense and will hopefully fix the test failure :)